packages/@okta/vuepress-site/docs/guides/registration-inline-hook/activate-and-enable/index.md

```diff
@@ -20,15 +20,52 @@ The Registration Inline Hook is now set up with a status of active.
 
 > **Note:** You can also setup an inline hook using an API. See [Inline Hooks Management API](/docs/reference/api/inline-hooks/#create-inline-hook) for further information.
 
+### Enabling the Registration Inline Hook
+
+The procedure to enable the Registration Inline Hook is dependent on the type of org you are using: Okta Identity Engine or Okta Classic. Follow the procedure for your specific org.
+
+After you enable the Registration Hook, you are ready to preview and test the example.
+
+#### Enabling the Registration Inline Hook in Okta Identity Engine
+
+<ApiLifecycle access="ie" />
+If you have an Okta Identity Engine org, you must [Enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm) to implement a Registration Inline Hook.
```

Comment:
```suggestion
If you have an Okta Identity Engine org, you must [enable and configure a profile enrollment policy](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/policies/create-profile-enrollment-policy-sr.htm) to implement a Registration Inline Hook.
```

Reply:
I guess I was maintaining the capitalization of the help topic I was referencing; however, I prefer it lower case.